rules/windows/process_creation/win_susp_registration_via_cscript.yml

```diff
@@ -0,0 +1,30 @@
+title: Suspicious Registration via cscript.exe
+id: 28c8f68b-098d-45af-8d43-8089f3e35403
+description: Detects when the registration of a VSS/VDS Provider as a COM+ application.
+status: experimental
+author: Austin Songer @austinsonger
+date: 2021/11/05
+references:
+- https://twitter.com/sblmsrsn/status/1456613494783160325?s=20
+- https://ss64.com/vb/cscript.html
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\cscript.exe'
+        CommandLine|contains|all:
+            - '-register'
+            - '\Windows Kits\10\bin\10.0.22000.0\x64'
```

Comment:
I think that it was not intended, that all three strings with "Windows Kits" in them have to appear in CommandLine to make this rule match, right? 

I guess it should be 
```
selection1:
    Image|endswith: '\cscript.exe'
    CommandLine|contains: '-register'
selection2:
    CommandLine|contains:
        - '\Windows Kits\10\bin\10.0.22000.0\x64'
        - '\Windows Kits\10\bin\10.0.19041.0\x64'
        - '\Windows Kits\10\bin\10.0.17763.0\x64'
    condition:
        selection1 and selection2